docs/configuration/config-file-reference.md

```diff
@@ -108,6 +108,10 @@ Where default_value is the value to use if the environment variable is undefined
 # The ruler_config configures the Cortex ruler.
 [ruler: <ruler_config>]
 
+# The configs_api_config configures the configs API used by the 'configs'
+# service to expose APIs to manage them.
+[configs_api: <configs_api_config>]
```

Comment:
From the configuration perspective, I'm a bit concerned having another root block config. Ideally we should have 1 root config block for each service (even if in the past some weird things have been done), so think it would be better adding the `NotificationsConfig` as part of the `ConfigDB` config.